gradle/dependencies.gradle

```diff
@@ -25,26 +25,33 @@ ext {
   defaultScalaVersions = [ '2.11', '2.12' ]
   // Available if -PscalaVersion is used. This is useful when we want to support a Scala version that has
   // a higher minimum Java requirement than Kafka. This was previously the case for Scala 2.12 and Java 7.
-  availableScalaVersions = [ '2.11', '2.12' ]
+  availableScalaVersions = [ '2.11', '2.12', '2.13' ]
 }
 
 // Add Scala version
 def defaultScala211Version = '2.11.12'
 def defaultScala212Version = '2.12.8'
+def availableScala213Version = '2.13.0-M5'
 if (hasProperty('scalaVersion')) {
   if (scalaVersion == '2.11') {
     versions["scala"] = defaultScala211Version
   } else if (scalaVersion == '2.12') {
     versions["scala"] = defaultScala212Version
-  } else {
+  } else if (scalaVersion == '2.13') {
+    versions["scala"] = availableScala213Version
+  }  else {
     versions["scala"] = scalaVersion
   }
 } else {
   versions["scala"] = defaultScala211Version
 }
 
 // Add base Scala version
-versions["baseScala"] = versions.scala.substring(0, versions.scala.lastIndexOf("."))
+if ( !versions.scala.startsWith('2.13.0-') ) {
+  versions["baseScala"] = versions.scala.substring(0, versions.scala.lastIndexOf("."))
+} else {
+  versions["baseScala"] = versions.scala
```

Comment:
Why does this need to be different? Is this just to make it clear it's not a valid final release with that version of scala or does something not work otherwise?

If we really do need this, you can make it generic by just checking for `-` in the Scala version string and we won't have to edit it either a) when we update to final 2.13.0 or b) if we want to test against future scala version milestones, i.e. 2.14+.

Reply:
@ewencp Makes sense. Actually, checking for a dash `-` only was one of interim solutions (now rebased).

Reply:
> Why does this need to be different? Is this just to make it clear it's not a valid final release with that version of scala or does something not work otherwise?

This hack (so to say) is added with intention to resolve **_scovarage_** artifacts versions builded against various `2.13-` Scala versions: 

```
  scoveragePlugin: "org.scoverage:scalac-scoverage-plugin_$versions.baseScala:$versions.scoverage",
  scoverageRuntime: "org.scoverage:scalac-scoverage-runtime_$versions.baseScala:$versions.scoverage",
```
(https://search.maven.org/search?q=g:org.scoverage%20a:scalac-scoverage-runtime_2.1*)

![image](https://user-images.githubusercontent.com/19467899/54521859-1fc7e380-496c-11e9-80b9-dcdae4a4cb86.png)

Reply:
New commit comments explain this (I hope). I will leave this review open (Maven central snapshot above should be visible, I reckon it explains a lot).

Reply:
@dejan2609 I think you should mention in the comment that pre-releases are not binary compatible with each other and that's the reason why libraries include the full Scala release string in their name for pre-releases. Focusing on cross-compilation doesn't explain the motivation and it's more of an implementation detail.

Reply:
Done.